tests/_support/Helper/Acceptance.php

```diff
@@ -77,23 +77,23 @@ public function addGutenbergBlock($I, $blockName, $blockProgrammaticName)
 		$I->seeElementInDOM('.block-editor-inserter__panel-content button.editor-block-list-item-' . $blockProgrammaticName);
 		$I->click('.block-editor-inserter__panel-content button.editor-block-list-item-' . $blockProgrammaticName);
 	}
-	
+
 	/**
-	 * Add the given block when adding or editing a Page, Post or Custom Post Type
-	 * in Gutenberg.
+	 * Check that the given block did not output any errors when rendered in the
+	 * Gutenberg editor.
 	 * 
-	 * @since 	1.9.6.9
+	 * @since 	1.9.7.4
+	 * 
+	 * @param 	AcceptanceTester 	$I 						Acceptance Tester.
+	 * @param 	string 				$blockName 				Block Name (e.g. 'ConvertKit Form')
 	 */
-	public function gutenbergAddBlock($I, $blockName, $blockProgrammaticName)
+	public function checkGutenbergBlockHasNoErrors($I, $blockName)
 	{
-		// Click Add Block Button.
-		$I->click('button.edit-post-header-toolbar__inserter-toggle');
+		// Wait a couple of seconds for the block to render.
+		sleep(2);
```

Comment:
Is there no other way to wait? Perhaps a loop that looks for a rendered block? A two-second test seems a bit slow.